src/libstd/panicking.rs

```diff
@@ -149,6 +149,18 @@ impl<'a> PanicInfo<'a> {
         self.payload
     }
 
+    /// Returns the payload as a slice if it is a `&'static str` or `String`.
```

Comment:
Note that this is a method of `PanicInfo`, which itself is already kinda niche. The payload here was originally given to the `panic!` macro. Does anyone use `panic!` with integers or vectors?